src/Suite.ts

```diff
@@ -22,6 +22,9 @@ async function writeTests() {
     });
 
     each(storiesByComponent, (stories, componentTitle) => {
+      if (!componentTitle.match(options.pattern)) {
```

Comment:
Can't imagine it makes a big difference, and I certainly haven't measured it, but it could make sense to use

```suggestion
      if (!options.pattern.test(componentTitle)) {
```

Which will return true or false, instead of null or the matches from the regex.